fetch_response.go

```diff
@@ -3,6 +3,8 @@ package sarama
 import (
 	"sort"
 	"time"
+
+	"bitbucket.org/ffxblue/bp-gateway/pkg/monolog"
```

Comment:
Internal project packaging, unrelated to Sarama, Need to remove all internal code references/use